Makefile

```diff
@@ -132,3 +132,46 @@ clean: ##@other Cleanup
 
 deep-clean: clean
 	rm -Rdf .ethereumtest/StatusChain
+
+release-branch: ##@release Create release branch and bump development version. Must be on develop branch
+	@if [ `git rev-parse --abbrev-ref HEAD` != develop ]; then \
```

Comment:
The approach is ok but we should change where these scripts are located. We should put them into a separate bash script file into a `ci/` directory or something like that and call them here instead of putting the code directly.

It should be possible to run these script files from the terminal and the effect should be the same.

Also, it would be better if this script could figure the current version from a git tag instead of `version.go`.

Reply:
The both points @adambabik says sounds reasonable - please add this changes and PR LGTM.